<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1338</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>931</width>
     <height>731</height>
    </rect>
   </property>
   <property name="title">
    <string>DISPLAY VIDEO</string>
   </property>
   <widget class="QLabel" name="image_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>901</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="frame_no_txt">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>750</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2"/>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>300</y>
     <width>127</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>SAVE DATA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="load_video_btn">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>300</y>
     <width>127</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>LOAD VIDEO</string>
   </property>
  </widget>
  <widget class="QSlider" name="frame_no_slider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>750</y>
     <width>911</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>30</y>
     <width>291</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>LABEL</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>290</width>
      <height>175</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="a_btn">
       <property name="text">
        <string>ADDRESS FRAME</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="a_txt"/>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="t_btn">
       <property name="text">
        <string>TOP FRAME</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="t_txt"/>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="i1_btn">
       <property name="text">
        <string>IMPACT START FRAME</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="i1_txt"/>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="i2_btn">
       <property name="text">
        <string>IMPACT  END FRAME</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="i2_txt"/>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="f_btn">
       <property name="text">
        <string>FINISH FRAME</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="f_txt"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
